editor/plugins/game_view_plugin.cpp

```diff
@@ -421,12 +431,16 @@ void GameView::_hide_selection_toggled(bool p_pressed) {
 	hide_selection->set_button_icon(get_editor_theme_icon(p_pressed ? SNAME("GuiVisibilityHidden") : SNAME("GuiVisibilityVisible")));
 
 	debugger->set_selection_visible(!p_pressed);
+
+	EditorSettings::get_singleton()->set_project_metadata("game_view", "hide_selection", p_pressed);
 }
 
 void GameView::_camera_override_button_toggled(bool p_pressed) {
 	_update_debugger_buttons();
 
 	debugger->set_camera_override(p_pressed);
+
+	EditorSettings::get_singleton()->set_project_metadata("game_view", "camera_override", p_pressed);
```

Comment:
This has no effect. The button is disabled and unpressed when project stops.
In either case, we shouldn't probably remember it for the same reason we don't remember input override.